src/DropdownMenu/DropdownButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {ChevronDownIcon} from '@primer/octicons-react'
+import Button, {ButtonProps} from '../Button/Button'
+
+export type DropdownButtonProps = ButtonProps
+
+export const DropdownButton = React.forwardRef<HTMLElement, React.PropsWithChildren<DropdownButtonProps>>(
+  ({children, ...props}: React.PropsWithChildren<DropdownButtonProps>, ref): JSX.Element => (
+    <Button ref={ref} {...props}>
+      {children}
+      <ChevronDownIcon />
```

Comment:
We try to avoid using the `chevron` icon for dropdown menus ([link](https://primer.style/design/ui-patterns/progressive-disclosure#progressive-disclosure-ui-patterns)). Let's use the `TriangleDownIcon` instead. Triangle-down is what we currently use for the dropdowns on dotcom. 

I also think adding a bit of space between the text and the icon would look a litter better.

```suggestion
      <StyledOcticon icon={TriangleDownIcon} ml={1} />
```

Reply:
Sounds good! Thanks for giving context and for providing a suggested alternative, @colebemis ❤️

Fixed in https://github.com/primer/components/pull/1135/commits/800a73878143ec3ea61080120ddf0585eb139ebf!